features/ASM000_Assemblies-information.feature

```diff
@@ -0,0 +1,21 @@
+@implementer-agreement
+@ASM
+@version1
+@E00020
+
+Feature: ASM000 - Assemblies information
```

Comment:
```suggestion
Feature: ASM000 - Composed elements
```

I like this. This is essentially what you've written in the final then step.

Reply:
Changed it in https://github.com/buildingSMART/ifc-gherkin-rules/pull/255/commits/6056105fa6493832381c41a50b742a8c0c54448e

Also at the feature filename, the PR name and the Jira ticket. 
It's also closer to the CT name (Element Composition)